### PR TITLE
QUARKUS-2349 - Fully support Java 17 and drop Java 11 for Native application

### PR DESCRIPTION
**Pull request by pjgg**
Fully support Java 17 and drop Java 11 for Native application - Test plan

### CODE REVIEW

File: QUARKUS-2349.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+## Scope of the testing
+
+Test development will focus on
+- Current native scenarios will be generated by GraalVM / Mandrel JDK17 docker images
+- GraalVM / Mandrel JDK11 must be deprecated (and removed) from all pipelines
+
+### Impact on testsuites and testing automation:
+
+- GraalVM/mandrel JDK17 docker images are going to be used to compile Quarkus native application regardless of whether the application runs on java 11 or 17
+- Native daily GitHub builds will cover GraalVM / Mandrel community JDK17 container images `ubi-quarkus-native-image:22.2-java17` and `ubi-quarkus-mandrel:22.2-java17`
```
Comment (rsvoboda) [inline QUARKUS-2349.md:21]:
```suggestion
- Native daily GitHub builds will cover GraalVM and Mandrel community JDK17 container images
  - `ubi-quarkus-native-image:22.2-java17` and `ubi-quarkus-mandrel:22.2-java17` now
  - targeting `ubi-quarkus-native-image:22.3-java17` and `ubi-quarkus-mandrel:22.3-java17` (when available) for Quarkus 2.13.z releases
```

**Review by rsvoboda — APPROVED** [review]

**Review by rsvoboda — APPROVED** [review]
Changes are needed

**Review by rsvoboda — CHANGES_REQUESTED** [review]
Changes are needed

File: QUARKUS-2349.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+ - [QUARKUS-728](QUARKUS-728)
+ - [QUARKUS-1397](QUARKUS-1397)
+
+## Scope of the testing
+
+Test development will focus on
+- Current native scenarios will be generated by GraalVM / Mandrel JDK17 docker images
+- GraalVM / Mandrel JDK11 must be deprecated (and removed) from all pipelines
+
```
[the remark this answers is not in the document]
Comment (rsvoboda) [reply QUARKUS-2349.md:17]: https://issues.redhat.com/browse/QUARKUS-2347 ?

[the remark this answers is not in the document]
Comment (pjgg) [reply QUARKUS-2349.md]: Yes is this one, I have added a reference directly in the TP

File: QUARKUS-2349.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+## Related test plans
+
+ - [QUARKUS-728](QUARKUS-728.md)
+ - [QUARKUS-1397](QUARKUS-1397.md)
+
+## Scope of the testing
+
+Test development will focus on
+- Upstream we run both mandrel and graalvm in some cases, in downstream we use mandrel only
```
Comment (rsvoboda) [inline QUARKUS-2349.md:15]:
```suggestion
- Upstream we run both Mandrel and GraalVM in some cases, in downstream we use Mandrel only
```

File: QUARKUS-2349.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+- Current native scenarios will be generated by GraalVM and Mandrel JDK17 native builder images
+- GraalVM and Mandrel JDK11 must be deprecated (and removed) from all pipelines
+- Product documentation need to be updated as well and remove old JDK references to Java 11
+    - https://issues.redhat.com/browse/QUARKUS-2347
+  
+### Impact on testsuites and testing automation:
+
+- GraalVM/mandrel JDK17 native builder images are going to be used to compile Quarkus native application regardless of whether the application runs on java 11 or 17
```
Comment (rsvoboda) [inline QUARKUS-2349.md:23]:
```suggestion
- GraalVM/Mandrel JDK17 native builder images are going to be used to compile Quarkus native application regardless of whether the application runs on java 11 or 17
```

**Review by rsvoboda — CHANGES_REQUESTED** [review]
Mandrel and GraalVM naming needs to be adjusted

**Review by rsvoboda — APPROVED** [review]

**Pull request merged**